setup-issue-templates/final-schedule.md

```diff
@@ -0,0 +1,3 @@
+Finalize the schedule once all content has been completed. 
```

Comment:
Is there any way to set that this issue is blocked? Maybe we could give it a "blocked" tag when creating it?